SharpPcap/LibPcap/LibPcapSafeNativeMethods.Windows.cs

```diff
@@ -308,5 +314,37 @@ static Windows()
         [DllImport(PCAP_DLL, CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
         internal extern static int pcap_sendqueue_transmit(IntPtr/*pcap_t * */p, ref pcap_send_queue queue, int sync);
         #endregion
+
+        #region Npcap specific
+
+
+        /// <summary>
+        /// Set the working mode of the interface p to mode. 
+        /// Valid values for mode are MODE_CAPT (default capture mode) 
+        /// and MODE_STAT (statistical mode). See the tutorial 
+        /// "\ref wpcap_tut9" for details about statistical mode.
+        /// Npcap specific method
+        /// </summary>
+        [DllImport(PCAP_DLL, CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        internal extern static int pcap_setmode(IntPtr/* pcap_t * */ p, int mode);
+
+        /// <summary>
+        /// changes the minimum amount of data in the kernel buffer that causes 
+        /// a read from the application to return (unless the timeout expires)
+        /// See http://www.winpcap.org/docs/docs_412/html/group__wpcapfunc.html#gab14ceacbf1c2f63026416dd73f80dc0d
```

Comment:
can we point to any tcpdump or libpcap docs for this? We should consider moving away from pointing at winpcap.org to prep for the domain going away at some point.

Reply:
Only mentioned in the immediate mode `pcap_set_immediate_mode` https://www.tcpdump.org/manpages/pcap_set_immediate_mode.3pcap.html

Reply:
yeah if that has similar or the appropriate content then lets use that url and keep moving away from the winpcap ones.